spec/reports/etas_organization_overlap_report_spec.rb

```diff
@@ -5,15 +5,11 @@
 require "reports/etas_organization_overlap_report"
 
 RSpec.describe Reports::EtasOrganizationOverlapReport do
-  let(:tmp_local) { "tmp_local_reports" }
-  let(:tmp_pers) { "tmp_pers_path" }
-  let(:tmp_rmt) { "tmp_remote_path" }
+  let(:tmp_local) { Settings.local_report_path }
```

Comment:
again here we can rely on the settings environment mechanism rather than hardcoding it. (Ultimately it might end up being better to use `Dir.mktmpdir`, though.)

Reply:
I would like to see a cleanup PR where we implement a good temp dir policy for all tests that generate output files.

Reply:
DEV-406